client/asset/btc/btc.go

```diff
@@ -599,23 +607,61 @@ func init() {
 	asset.Register(BipID, &Driver{})
 }
 
-// baseWalletConfig is the config params for the base wallet.
+// baseWalletConfig is the validated, unit-converted, user-configurable wallet
+// settings.
 type baseWalletConfig struct {
 	fallbackFeeRate  uint64 // atoms/byte
 	feeRateLimit     uint64 // atoms/byte
 	redeemConfTarget uint64
 	useSplitTx       bool
 }
 
+func readBaseWalletConfig(walletCfg *WalletConfig) (*baseWalletConfig, error) {
```

Comment:
Since you're moving this function, I'd prefer if you put it above `readRPCWalletConfig`, before its first use.